### ISSUE TITLE
Add easy login for API backends

### ISSUE THREAD

**Issue by Emmo00** [FahadYousafMahar/slash-login#2]
I see the current implementation provides an easy way to log into you app with web routes but it might be nice to add functionality for easy API route login

What do you think of this? @FahadYousafMahar ... I'd be happy to be assigned this if it's something you want to add to this package

**Comment by FahadYousafMahar** [comment]
Once you publish the configuration. 
`php artisan vendor:publish --tag="slash-login-config"`
You can change the guard from **web** to **api**.
If that solves your case, that would be fine. Otherwise, you can provide more details of how you're using it and I'll look into it further.

**Comment by Emmo00** [comment]
You're right, but the behaviour on an api server is a little different from what we have now:

- we don't need to push anything to the session
- A JSON response is expected with a `token` (auth token) field (not a redirect)

**Comment by FahadYousafMahar** [comment]
Alright. 
You're welcome to contribute to it.
